examples/tools/scene_viewer/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod camera_controller_plugin;
+mod scene_viewer_plugin;
+
+pub use camera_controller_plugin::*;
```

Comment:
I don't like these `pub use *` invocations here. I don't think they make the structure clearer. I would rather just make the modules pub.